fix(docs-improvements): Fix entity examples (#5381)
Kong/developer.konghq.com: app/_includes/components/entity_example/format/ui.md

1. Navigate to your Gateway instance:

>   * In Konnect, open **API Gateway** from the sidebar, then choose a control plane.

Reviewer: ```suggestion
  * In the Konnect sidebar, click **API Gateway** and then choose a control plane.
```
Or `* In the Konnect sidebar, click **API Gateway** > **Gateways** and then choose a control plane.` if you're using the new UI view.

Author: Ahh I forgot there's an extra link in the new one now. I'll fix this.

1. Navigate to your Gateway instance:
  * In Konnect, open **API Gateway** from the sidebar, then choose a control plane.
  * In Kong Manager, select your Workspace.
1. Navigate to Consumers:

>   * In Konnect, click the **Consumers** tab inside your control plane.

Reviewer: Same edit from above would apply here
